### DISCUSSION TITLE
URL does not match quarkus.http.root-path in dev mode

### DISCUSSION THREAD

**Discussion by faermanj** [quarkusio/quarkus#40239] [Community]
So, not sure if this is a bug, what do you think?
URL does not seem to use the root path (quarkus.http.root-path), generating wrong links, as shown in the image below
/pvt/sandbox should be /user/cfp/pvt/sandbox in the URL (generated link)
![image](https://github.com/quarkusio/quarkus/assets/356476/a36a50d9-40e1-437e-88ed-29953769cdf6)

**Comment by gsmet** [comment]
I created an issue from the discussion here: https://github.com/quarkusio/quarkus/issues/40240 .

I think it's a bug or at minima something that requires an investigation.

Thanks for reporting!
